### PR TITLE
Document how to use custom certs with kubeadm

### PR DESCRIPTION
**Pull request by jamiehannaford**
Fixes https://github.com/kubernetes/kubeadm/issues/239

cc/ @luxas @calvix

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4113)

### CODE REVIEW

File: docs/admin/kubeadm.md [hunk 1]
```diff
@@ -446,6 +447,22 @@ EOF
...
+
+To do so, you must place them in whatever directory is specified by the
+`--cert-dir` flag or `CertificatesDir` configuration file key. By default this
+is `/etc/kubernetes/pki`.
+
+If a given certificate and private key pair both exist, kubeadm will skip the
+generation step and those files will be validated and used for the prescribed
+use-case.
+
+This means the user can, for example, prepopulate `/etc/kubernetes/pki/ca.{crt,key}`
```
Comment (ahmetb) [inline docs/admin/kubeadm.md]: I suggest expanding this as

> `/etc/kubernetes/pki/ca.crt` and `/etc/kubernetes/pki/ca.key`.

File: docs/admin/kubeadm.md [hunk 2]
```diff
@@ -23,7 +23,8 @@ following steps:
 
 1. kubeadm generates a self-signed CA to provision identities for each component
    (including nodes) in the cluster.  It also generates client certificates to
-   be used by various components.
+   be used by various components. If the user has provided their own CA by
+   dropping it in the cert directory, this step is skipped.
```
Comment (ahmetb) [inline docs/admin/kubeadm.md]: I suggest perhaps referring to the "cert directory" as 

> cert directory (configured via `--cert-dir`, by default `/etc/kubernetes/pki`) ...

File: docs/admin/kubeadm.md [hunk 3]
```diff
@@ -446,6 +447,22 @@ EOF
 
 Now `kubelet` is ready to use the specified CRI runtime, and you can continue with `kubeadm init` and `kubeadm join` workflow to deploy Kubernetes cluster.
 
+## Using custom certificates
+
+By default kubeadm will generate all the certificates needed for a cluster to run.
+The user can override this behaviour by providing your own certificates.
```
Comment (chenopis) [inline docs/admin/kubeadm.md]: Stick w/ second-person point of view: "The user" -> "You"

**Comment by jamiehannaford** [comment]
@chenopis @ahmetb Thanks for the review. I've incorporated your feedback, PTAL :)

**Pull request merged**